Mathlib/CategoryTheory/ChosenFiniteProducts.lean

```diff
@@ -118,6 +118,13 @@ lemma hom_ext {T X Y : C} (f g : T ⟶ X ⊗ Y)
     f = g :=
   (product X Y).isLimit.hom_ext fun ⟨j⟩ => j.recOn h_fst h_snd
 
+@[reassoc, simp]
```

Comment:
```suggestion
-- Similarly to `CategoryTheory.Limits.prod.comp_lift`, we do not make the `assoc` version a simp lemma
@[reassoc, simp]
```

The lemma you copied, `CategoryTheory.Limits.prod.comp_lift`, has the bad explanation `-- Making the reassoc version of this a simp lemma seems to be more harmful than helpful.`. I suggest adding this comment here to point readers that are wondering about this to that lemma, so that there is at least some explanation.

Reply:
Right, I copy-pasted this a bit too hastily perhaps.